### DISCUSSION TITLE
Command /claimfirespread on/off

### DISCUSSION THREAD

**Discussion by NullCase** [GriefPrevention/GriefPrevention#688] [Ideas]
**As** a player/server owner
**I want** claim owners / claim managers to have the option to turn fire spread on.
**Because** It's their land claim and fire spread is a popular request.

I can put some $$ up to fund this as usual. Also happy to test and provide feeback. Also open to changes to the command name.

For an owner/manager typing `/firespread on` when standing in a landclaim with owner perms OR in a subclaim with managment perms...

but if you wanted to keep it consistent with /claimexplosions
then /claimfirespread seems like a fine option to me.

**Comment by ShaneBeee** [comment]
If you want, for the time being, you can use [**GriefPrevention Flags**](https://www.spigotmc.org/resources/griefprevention-flags.55773/) there's a [**NoFireSpread**](https://www.spigotmc.org/resources/griefprevention-flags.55773/) flag that was added for this very reason.

**Comment by Weasel-Beans** [comment]
> If you want, for the time being, you can use [**GriefPrevention Flags**](https://www.spigotmc.org/resources/griefprevention-flags.55773/) there's a [**NoFireSpread**](https://www.spigotmc.org/resources/griefprevention-flags.55773/) flag that was added for this very reason.

Doesn't that only work to disable fire-spread though? I believe if fire-spread is disabled by GriefPrevention, then there's no way to have fire enabled in claimed areas, which is the majority of cases as enabling global fire-spread is just asking for griefing/lag

**Comment by NullCase** [comment]
griefing, lag... yeah. and so about one year ago when a crew set our forests on fire, that lag was defintely a consideration. Still, I want more like that.

in particular it seems like /claimexplosions is a really great option for claim owners. Adding all options so claim owners can turn off whatever protections are in place... I realize now this won't be for everyone. Letting claim owners turn off whatever non-vanilla protections are in place is an option that I'd like to offer.

**Comment by RoboMWM** [comment]
Agreed, and is the plan for v20 (still trying to find a good name other than "natural grief").

**Comment by NullCase** [comment]
For v20, do you have all of your funding lined up? As always I have to be careful about spending, but it looks like'll be able to put some $ in for v20, especially if its particular features (like in this case).

**Comment by RoboMWM** [comment]
#63 has most of the details right now on v20. If anything I need testers and feedback much more than funding right now.

**Comment by NullCase** [comment]
Hey @RoboMWM 

Are the updates you're pushing to [appveyor](https://ci.appveyor.com/project/RoboMWM39862/griefprevention/build/artifacts) for GP v20?

Edit: it looks like no, but in that case...Whats the easiest way to get the .jar for v20? To start doing testing/feedback.

**Comment by RoboMWM** [comment]
They are indeed pushing to appveyor. It's just lumped in with master as well. This was the last successful build (apparently it's not passing the new tests I wrote despite them working fine on my machine) https://ci.appveyor.com/project/RoboMWM39862/griefprevention/builds/27588138/artifacts
